docs/resources/techresources/gitgithub.md

```diff
@@ -113,4 +113,4 @@ release its contents into your Production runtime environment.
 
 - [GitHub Guides](https://guides.github.com/)
 - [GitHub Help](https://help.github.com/)
-- [tryGit](https://try.github.io/levels/1/challenges/1)
\ No newline at end of file
+- [tryGit](https://try.github.io/levels/1/challenges/1)
```

Comment:
these 2 lines look the same to me

Reply:
Absolutely, @cherylli. I don't know why GitHub marked it as a change. I didn't touch any of those lines.